src/libtiled/tilelayer.h

```diff
@@ -266,7 +268,7 @@ class TILEDSHARED_EXPORT TileLayer : public Layer
      * are rotated within the layer, and the tiles themselves are rotated. The
      * dimensions of the tile layer are swapped.
      */
-    void rotateHexagonal(RotateDirection direction);
+    void rotateHexagonal(RotateDirection direction, Tiled::Map::StaggerIndex staggerIndex, Tiled::Map::StaggerAxis staggerAxis);
```

Comment:
You're in the `Tiled` namespace here, so you don't need the `Tiled::` qualification.